documentation/release_notes.rst

```diff
@@ -95,6 +95,8 @@ New in This Release
 ^^^^^^^^^^^^^^^^^^^
 - Incorrect results may be produced with in-place scans using ``unseq`` and ``par_unseq`` policies on
   CPUs with the Intel® C++ Compiler 2021.8.
+- Incorrect results may be produced with group reduce operation through ``std::reduce``
```

Comment:
Could you please move it under "New in 2022.3.0" section? As it is the version to be released, so the users will see its' release notes.

Reply:
Fixed